index.js

```diff
@@ -130,6 +130,12 @@ Transformer.prototype.can = function (method) {
 
 Transformer.prototype.compile = function (str, options) {
   if (!this.can('compile')) {
+    if (this.can('render')) {
+      var _this = this;
+      return tr.normalizeFn(function (locals) {
+        return tr.normalize(_this._tr.render(str, options, locals));
```

Comment:
I don't think the dependency is handled correctly. `normalize` returns `{ body, dependencies }`, so `normalizeFn` will return a `{ fn: function (locals) -> { body, dependencies }, dependencies: [] }`, while `compile` is supposed to return `{ fn: function (locals) -> body, dependencies }`.

Now, because we don't have a `compile`, we don't know the `dependencies` beforehand, so we have to leave it as `[]`. We should still return a `{ fn: function (locals) -> body, dependencies: [] }` however.

I think it should be like this:

```js
      return {
        fn: function (locals) {
          return tr.normalize(_this._tr.render(str, options, locals)).body;
        },
        dependencies: []
      };

      /// Above is equivalent to
      return tr.normalizeFn(function (locals) {
        return tr.normalize(_this._tr.render(str, options, locals)).body;
      });
```

@ForbesLindesay can you double check?